src/properties.rs

```diff
@@ -10,6 +12,7 @@ pub struct FileProperties {
 	pub(crate) sample_rate: Option<u32>,
 	pub(crate) bit_depth: Option<u8>,
 	pub(crate) channels: Option<u8>,
+	pub(crate) channel_mask: Option<ChannelMask>,
```

Comment:
This'll need a getter like the other fields

Reply:
https://github.com/Serial-ATA/lofty-rs/pull/155/commits/6672b3fe8303f269fe12999f04dcf21266e47de7